policyengine_us/parameters/gov/states/mo/tax/income/rates.yaml

```
@@ -123,6 +126,11 @@ metadata:
       href: https://revisor.mo.gov/main/OneSection.aspx?section=143.011&bid=51511&hl=
     - title: 2024 Missouri Withholding Tax Formula
       href: https://dor.mo.gov/forms/Withholding%20Formula_2024.pdf#page=2
+    - title: Missouri Individual Income Tax Changes 2024
+      href: https://dor.mo.gov/taxation/individual/tax-types/income/year-changes/
+    - title: 2025 Missouri Withholding Tax Formula
+      href: https://dor.mo.gov/forms/Withholding%20Table%20-%20Weekly_2025.pdf
```

As this is where the top rate is specified
```suggestion
      href: https://dor.mo.gov/forms/Withholding%20Table%20-%20Weekly_2025.pdf#page=2
```